libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/TwoPhaseValidation.hs

```diff
@@ -783,11 +784,11 @@ utxoStEx6 pf = smartUTxOState (utxoEx6 pf) (Coin 0) (Coin 5) def
 --  Example 7: Process a MINT transaction with a succeeding Plutus script.
 -- =============================================================================
 
-mintEx7 :: forall era. (Scriptic era, HasTokens era) => Proof era -> Value era
+mintEx7 :: forall era. (Scriptic era, HasTokens era) => Proof era -> MultiAsset (Crypto era)
 mintEx7 pf = forge @era 1 (always 2 pf)
 
-outEx7 :: (HasTokens era, EraTxOut era, Scriptic era) => Proof era -> TxOut era
-outEx7 pf = newTxOut pf [Address (someAddr pf), Amount (mintEx7 pf <+> inject (Coin 995))]
+outEx7 :: (HasTokens era, EraTxOut era, Scriptic era, Value era ~ MaryValue (Crypto era)) => Proof era -> TxOut era
```

Comment:
do you know why we need `Value era ~ MaryValue (Crypto era)` now?

Reply:
That's a good question.  I accepted this sort of mechanically (because the `Amount`  field defined for tests takes a Value), but I haven't questioned this. Looking to see if it's indeed necessary, thanks for the suggestion

Reply:
thanks! I was mostly curious...